.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12, 14]
+    name: Node ${{ matrix.node }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use node ${{ matrix.node }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install
+        run: npm install
+      - name: Test
+        if: ${{ matrix.node != 14}}
```

Comment:
This line can be removed now, right?

Reply:
I don't recall what it was used for.

Reply:
I forgot to remove it from the previous commit.

I've taken it out now.